event.go

```diff
@@ -0,0 +1,47 @@
+package instana
+
+type EventData struct {
+	Title    string `json:"title"`
+	Text     string `json:"text"`
+	Duration int    `json:"duration"`
+	Severity int    `json:"severity"`
+	Plugin   string `json:"plugin,omitempty"`
+	ID       string `json:"id,omitempty"`
+	Host     string `json:"host,omitempty"`
+}
+
+const (
+	ServicePlugin = "com.instana.forge.connection.http.logical.LogicalWebApp"
+	ServiceHost   = ""
+)
+
+func SendDefaultServiceEvent(title string, text string, severity int) {
+	SendServiceEvent(sensor.serviceName, title, text, severity)
+}
+
+func SendServiceEvent(service string, title string, text string, severity int) {
+	sendEvent(&EventData{
```

Comment:
the frequency with which events arrive is not expected to be as high as traces or metrics, plus events API on the agent is generic and doesn't need any dances around pid in different scenarios.